RevenueCatUI/CustomerCenter/Views/RestorePurchasesAlert.swift

```diff
@@ -75,7 +79,21 @@ struct RestorePurchasesAlert: ViewModifier {
                     return Alert(title: Text(""),
                                  message: Text("We couldn’t find any additional purchases under this account. \n\n" +
                                                "Contact support for assistance if you think this is an error."),
-                                 dismissButton: .default(Text("Dismiss")) {
+                                 primaryButton: .default(Text(localization.commonLocalizedString(for: .contactSupport)),
+                                                         action: {
+                        if let supportInformation = self.supportInformation {
+                            let subject = self.localization.commonLocalizedString(for: .defaultSubject)
+                            let body = self.localization.commonLocalizedString(for: .defaultBody)
```

Comment:
I think it might be good in future iterations to have this body include some info on the device/user, but we can do this in follow-up PRs 👍

Reply:
Yes! I will do that this week, I think it will be super useful